cost/turbonomics/scale_virtual_machines_recommendations/aws/turbonomics_scale_virtual_machines.pt

```diff
@@ -0,0 +1,396 @@
+            return
+          });
+        } else if (vm.provider === "GCP Project") {
+          Object.keys(vm.vendorIDValues).forEach(function(key) {
+            vm.resourceID = vm.vendorIDValues[key]
+            return
+          });
+        } else {
+          Object.keys(vm.vendorIDValues).forEach(function(key) {
+            vm.resourceID = vm.vendorIDValues[key]
+            return
+          });
+        }
+        vm.service = provider_service[vm.provider]
+        savingsString = ""
+        if (typeof vm.savings === "undefined" || vm.savings === null || vm.savings === "" || isNaN(vm.savings)) {
+          vm.savings = 0.0
+        }
+        vm.savings = (Math.round(vm.savings * 730 * 1000 * (vm.uptime/100)) / 1000)
+        monthlySavings = monthlySavings + vm.savings
+        instances.push(vm)
+      }
+    })
+    message = ""
+    if (instances.length != 0) {
+      pretty_savings = instances[0].savingsCurrency + ' ' + formatNumber(monthlySavings.toFixed(2), ",")
+      message = "The total estimated monthly savings are " + pretty_savings + '.'
+    }
+    result = {
+      'instances': instances,
+      'message': message
+    }
+EOS
+end
+
+###############################################################################
+# Policy
+###############################################################################
+
+policy "pol_turbonomics_scale_vms" do
+  validate $ds_filtered_turbonomics_recommendations do
+    summary_template "{{ rs_project_name }} (Account ID: {{ rs_project_id }}): {{ len data.instances }} Turbonomic Scalable Virtual Machines found"
+    detail_template <<-EOS
+## Turbonomics Scale Virtual Machines POC
+{{data.message}}
+EOS
+    check eq(size(val(data, "instances")), 0)
+    export "instances" do
+      field "accountID" do
+        label "Account ID"
+      end
+      field "accountName" do
+        label "Account Name"
+      end
+      field "resourceID" do
+        label "Resource ID"
+      end
+      field "resourceName" do
+        label "Resource Name"
+      end
+      field "savingsCurrency" do
+        label "Savings Currency"
+      end
+      field "savings" do
+        label "Savings"
+      end
+      field "details" do
+        label "Recommendation Details"
+      end
+      field "uptime" do
+        label "Virtual Machine Uptime Percentage"
+      end
+      field "currentResourceType" do
+        label "Current Compute Tier"
+        path "resourceType"
+      end
+      field "newResourceType" do
+        label "New Compute Tier"
+      end
+      field "region" do
+        label "Region"
+      end
+      field "service" do
+        label "Service"
+      end
+      field "resourceGroup" do
+        label "Resource Group"
+      end
+      field "tags" do
+        label "Tags"
+      end
+      field "provider" do
+        label "Cloud Provider"
+      end
+      field "createdTime" do
+        label "Created Time"
+      end
+      field "id" do
```

Comment:
I think this field is already added on line 331.

Reply:
You were right. When testing I saw that field repeated. Thanks a lot!